### DISCUSSION TITLE
Relation picker displays ID only

### DISCUSSION THREAD

**Discussion by shsf1382hAcKeR** [pocketbase/pocketbase#5083] [Q&A]
![image](https://github.com/pocketbase/pocketbase/assets/53711984/bd3a105b-c3b7-4962-88a8-9d8d47d997f9)

Is there any way to display a custom field for the relation? It would really help with the picking process.

I am using v0.22.13 and I saw the following fix in v0.22.11:
- Load the full record in the relation picker edit panel (#4857)

Idk if this fix was related to my problem or not but I am still having the issue on v0.22.13

**Comment by shsf1382hAcKeR** [comment]
I forgot to mention that I am not using the prebuilt executables and I am using my own build (extended with GO)

**Reply by ganigeorgiev** [reply]
Your issue is not related to the linked bugfix.
Most likely you collection doesn't have an explicit "Presentable" field OR have another `relation` field marked as "Presentable" which currently will display just the `relation` field value, aka. the id (see https://github.com/pocketbase/pocketbase/issues/3666).

The relations representation will change in the future and most likely the "Presentable" option will be removed completely with the planned Admin UI [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355), but for now the server-side changes are with higher priority.
